website_and_docs/content/documentation/webdriver/elements/information.zh-cn.md

```diff
@@ -6,23 +6,13 @@ description: >
   元素相关的知识.
 ---
 
-您可以查询有关特定元素的许多详细信息.
+您可以查询有关特定元素的许多详细信息。
 
 ## 是否显示
 
-此方法用于检查连接的元素是否正确
-显示在网页上. 返回一个 `Boolean` 值, 
-如果连接的元素显示在当前的浏览器上下文中, 则为True
-否则返回false.
+此方法用于检查连接的元素是否正确显示在网页上. 返回一个 `Boolean` 值，如果连接的元素显示在当前的浏览器上下文中，则为True，否则返回false。
 
-此功能[提及于](https://w3c.github.io/webdriver/#element-displayedness), 
-但是未定义于w3c规范中, 
-因为[无法覆盖所有潜在条件](https://www.youtube.com/watch?v=LAD_XPGP_kk).
-因此，Selenium不能期望驱动程序
-直接实现这种功能，现在依赖于
-直接执行大量JavaScript函数. 
-此函数界定了元素树中的许多本身属性以及关系, 
-并以值的形式返回.
+此功能[于W3C规范中提及](https://w3c.github.io/webdriver/#element-displayedness)，但由于[无法覆盖所有潜在条件](https://www.youtube.com/watch?v=LAD_XPGP_kk)而无法定义。因此，Selenium不能期望驱动程序直接实现这种功能，现在依赖于直接执行大量JavaScript函数。这个函数对一个元素的性质和在树中的关系做了许多近似的判断，以返回一个值。
```

Comment:
This is toooooo long in a line, please refer to contribution guide for the translation rule
https://www.selenium.dev/documentation/about/contributing/#step-4-commit

Reply:
Well, since it's a markdown file, I'm afraid that if you split between lines, there will be extra spaces which make it somehow weird to read.

The previous page translation was also weird, I've changed some wording to make it more natural with what Simplified Chinese readers more like(maybe I should have split this into more PRs 😄). Also, the preivous version has used a lot of English punctuation marks, which seemed also weird with Simplified Chinese context.

Reply:
```suggestion
此功能[于W3C规范中提及](https://w3c.github.io/webdriver/#element-displayedness)，但由于[无法覆盖所有潜在条件](https://www.youtube.com/watch?v=LAD_XPGP_kk)而无法定义。
因此，Selenium不能期望驱动程序直接实现这种功能，现在依赖于直接执行大量JavaScript函数。这个函数对一个元素的性质和在树中的关系做了许多近似的判断，以返回一个值。
```

maybe we can split like this

Reply:
> Well, since it's a markdown file, I'm afraid that if you split between lines, there will be extra spaces which make it somehow weird to read. A workaround maybe spliting between sentences(or full stops).
> 
> The previous page translation was also weird, I've changed some wording to make it more natural with what Simplified Chinese readers more like(maybe I should have split this into more PRs 😄). Also, the preivous version has used a lot of English punctuation marks, which seemed also weird with Simplified Chinese context.

This is the rule of Selenium Document. Do not make one line too long. You can refer above given link to check the detail.
You can try to build website locally with Hugo to diff any external line later.

About the punctuation mark, it's no impacting on reader,  but make more difference than original content. No need to change an existing page with all changed, you can just update the changed parts.

Reply:
ok, I will revert that part of commit then